server/api/src/rsDataObjRsync.cpp

```diff
@@ -11,10 +11,16 @@
 #include "irods/rsDataObjCopy.hpp"
 
 #include "irods/irods_resource_redirect.hpp"
+#include "irods/irods_logger.hpp"
 
 #define IRODS_FILESYSTEM_ENABLE_SERVER_SIDE_API
 #include "irods/filesystem.hpp"
 
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+using log_api = irods::experimental::log::api;
```

Comment:
Let's put these in an anonymous namespace so they are private within the binary.